frontend/public/components/app-contents.tsx

```diff
@@ -465,7 +465,7 @@ const AppContents_: React.FC<AppContentsProps> = ({ activePerspective, flags })
             exact
             loader={() =>
               import(
-                './monitoring/alert-manager-receiver-forms' /* webpackChunkName: "monitoring" */
+                './monitoring/receiver-forms/alert-manager-receiver-forms' /* webpackChunkName: "monitoring" */
```

Comment:
I'd suggest a separate webpack chunk for receiver forms

Reply:
So change `/* webpackChunkName: "monitoring" */` to `/* webpackChunkName: "receiver-forms" */` ?

Reply:
fixed